src/content/app/genome-browser/hooks/useBrowserRouting.test.tsx

```diff
@@ -15,42 +15,38 @@
  */
 
 import React, { createContext, useContext } from 'react';
-import { render, act } from '@testing-library/react';
+import { render, act, waitFor } from '@testing-library/react';
+import { configureStore } from '@reduxjs/toolkit';
 import { Provider } from 'react-redux';
-import thunk from 'redux-thunk';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
 import { MemoryRouter, Routes, Route, useLocation } from 'react-router';
-import configureMockStore from 'redux-mock-store';
 import set from 'lodash/fp/set';
 
-import * as genomeActions from 'src/shared/state/genome/genomeSlice';
-import * as browserGeneralActions from '../state/browser-general/browserGeneralSlice';
+import createRootReducer from 'src/root/rootReducer';
+
+import restApiSlice from 'src/shared/state/api-slices/restSlice';
 
 import useBrowserRouting from './useBrowserRouting';
 
 import { createSelectedSpecies } from 'tests/fixtures/selected-species';
-import { createGenomeCategories } from 'tests/fixtures/genomes';
 
 // NOTE: scary stuff, but if you prefix function name with the word "mock",
 // jest will allow passing them to the factory function of jest.mock
 const mockChangeFocusObject = jest.fn();
 const mockChangeBrowserLocation = jest.fn();
 
-jest.mock(
-  'src/content/app/genome-browser/state/browser-general/browserGeneralSlice.ts',
-  () => ({
-    setActiveGenomeId: jest.fn(() => ({ type: 'setActiveGenomeId' })),
-    setDataFromUrlAndSave: jest.fn(() => ({ type: 'setDataFromUrlAndSave' }))
-  })
-);
+const mockGenomeSearchApiBaseUrl = 'http://genome-search-api';
+const mockGenomeBrowserObj = {};
 
-jest.mock('src/shared/state/genome/genomeSlice', () => ({
-  fetchGenomeData: jest.fn(() => ({ type: 'fetchGenomeData' }))
+jest.mock('config', () => ({
+  genomeSearchBaseUrl: 'http://genome-search-api'
```

Comment:
```suggestion
  genomeSearchBaseUrl: mockGenomeSearchApiBaseUrl
```

Reply:
That actually won't work, I don't think. I tried it initially. I still don't understand jest's rules about magically named constants. They seem to work if you created a factory function that would return another function which would use the magic constant, like so: 

```
jest.mock('config', () => ({
  genomeSearchBaseUrl: () => mockGenomeSearchApiBaseUrl
})
```

But they don't seem to work when the factory function actually returns something that immediately uses the magic constant, like in your example.

In any case, these jest mocks are a disaster waiting to happen 🙈 I suspect it will all collapse when Jest finally gets ready to be migrated to use ES6 imports.